website/docs/zh/config/output/filename.mdx

```diff
@@ -55,6 +57,7 @@ const prodDefaultFilename = {
 - `font`：表示字体文件的名称。
 - `image`：表示非 SVG 图片的名称。
 - `media`：表示视频等媒体资源的名称。
+- `assets`：表示除了上述类型之外，其他命中 `asset/resource` 的文件名称。
```

Comment:
How about linking to the Asset Module documentation?

The same as `output.assetModuleFilename`

![image](https://github.com/user-attachments/assets/aef6557c-6c98-4c45-adcd-2838a2501ee4)


```suggestion
- `assets`：表示除了上述类型之外，其他命中 [Asset Modules](https://rspack.dev/guide/features/asset-module) 规则的文件名称。
```